kit/src/main/java/com/oracle/javafx/scenebuilder/kit/editor/panel/inspector/editors/I18nStringEditor.java

```diff
@@ -121,6 +138,8 @@ public Object getValue() {
         String val = textNode.getText();
         if (i18nMode) {
             val = new PrefixedValue(PrefixedValue.Type.RESOURCE_KEY, val).toString();
+        } else if (exprMode) {
+        	val = new PrefixedValue(PrefixedValue.Type.BINDING_EXPRESSION, val).toString();
```

Comment:
Make sure to sure 'spaces' instead of 'tabs'. There are multiple other places where tabs have been introduced.